.github/workflows/ci.yml

```diff
@@ -31,5 +19,7 @@ jobs:
         id: ci
         uses: gazebo-tooling/action-gz-ci@noble
         with:
+          # codecov-enabled: true
```

Comment:
Just checking, is this on purpose?

Reply:
yes, this is intentional. codecov currently fails on Noble (see https://github.com/gazebosim/gz-cmake/pull/497), so it's disabled for all the Jetty branches on Noble